.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -28,11 +28,14 @@ examples of each step, assuming vX.Y.Z is being cut.
   - [ ] `xp/getting-started-with-aws-with-vpc`
   - [ ] `xp/getting-started-with-azure`
   - [ ] `xp/getting-started-with-gcp`
+- [ ] Confirm the full set of patch versions that will be released and promote them from lowest to highest, so the highest version is the last to be promoted (e.g. `v1.12.2` should be promoted after `v1.11.3`)
```

Comment:
Maybe mentioning the reason here could be helpful?

Reply:
```suggestion
- [ ] Confirm the full set of patch versions that will be released and promote them from lowest to highest, so the highest version is the last to be promoted (e.g. `v1.12.2` should be promoted after `v1.11.3`, otherwise the promote workflow will overwrite the latest release with the last one).
```

Reply:
@jbw976 any thought here?

Reply:
Good idea @phisco! i included similar language to your suggestion to make this clarification 🤓